spec/spec_helper.rb

```diff
@@ -37,6 +37,7 @@
 
 RSpec.configure do |config|
   config.include TableMethods
+  config.filter_run_when_matching :focus
```

Comment:
[This](https://relishapp.com/rspec/rspec-core/v/3-10/docs/filtering/filter-run-when-matching) allows us to focus on `describe/context/it`s. I can revert this change too.